### PR TITLE
readme update: screen shots and status

### PR DESCRIPTION
**Pull request by gucio321**
Hi there,
that's README.md update for december.

### CODE REVIEW

**Comment by ThomasChr** [comment]
nice!
Good to keep that updated!

File: README.md [hunk 1]
```diff
@@ -25,7 +25,8 @@ ALL OTHER TRADEMARKS ARE THE PROPERTY OF THEIR RESPECTIVE OWNERS.
 
 ## Status
 
-At the moment (october 2020) the game starts, you can select any character and run around Act1 town.
+At the moment (december 2020) the game starts, you can select any character and run around Act1 town.
+You can also open any of of the game panel.
```
Comment (essial) [inline README.md:29]: Please change to "You can also open any of the game's panels."

**Comment by essial** [comment]
Looks good. Once you correct the grammar error I'll merge, thanks so much!

**Comment by gucio321** [comment]
thanks @essial. Now should be OK.

**Pull request merged**